packages/next/components/chain/chainIcon.js

```diff
@@ -8,8 +8,16 @@ import Moonriver from "public/imgs/icons/chain/moonriver.svg";
 import Polkadex from "public/imgs/icons/chain/polkadex.svg";
 import Interlay from "public/imgs/icons/chain/interlay.svg";
 import { CHAINS } from "../../frontedUtils/consts/chains";
+import styled from "styled-components";
+import Tooltip from "../tooltip";
+import { capitalize } from "frontedUtils";
 
-function ChainIcon({ chainName, size = 24 }) {
+const Wrapper = styled.span`
```

Comment:
use `span` as a wrapper is not recommended, `div`, `li`, `form` ... native block elements are preferred.

Reply:
I referenced `identityIcon.js`, that's why I use `span` as Wrapper